src/wallet/typings/Balance.d.ts

```diff
@@ -0,0 +1,44 @@
+import { Fixed8 } from '../../utils';
+import { Transaction } from '../../transactions/index'
+import { AssetBalance } from './AssetBalance';
+
+/** Describes the coins found within an Account. Look up various balances through its symbol. For example, NEO or GAS. */
+export class Balance {
+  constructor(bal?: Balance)
+
+  /** The address for this Balance */
+  address: string
+
+  /** The network for this Balance */
+  net: 'MainNet' | 'TestNet'
```

Comment:
This can be any string too.

Reply:
updated to use same type from `api.core`